docker-extension-examples/examples/docker-deployment-without-annotations/docker_deployment_without_annotations.bal

```diff
@@ -0,0 +1,21 @@
+import ballerina/http;
+import ballerina/log;
+//Adding the import as `ballerina/docker as _` will generate the Docker image and Dockerfile for the `helloWorld` service.
+import ballerina/docker as _;
+
+listener http:Listener helloWorldEP = new(9090);
+
+@http:ServiceConfig {
+    basePath: "/helloWorld"
+}
+service helloWorld on helloWorldEP {
+    resource function sayHello(http:Caller outboundEP, http:Request request) {
+        http:Response response = new;
+        response.setTextPayload("Hello World from Docker ! \n");
+        var responseResult = outboundEP->respond(response);
+        if (responseResult is error) {
```

Comment:
For hello world samples, check and return an error instead of logging it. 
e.g
resource function sayHello(http:Caller outboundEP,http:Request request) returns error? {
      check outboundEP->respond("Hello World!");
  }

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerina-docker/pull/445/commits/8e6f027b0cb2bf60326dd9a039aca7656e045d24 and https://github.com/ballerina-platform/module-ballerina-docker/pull/445/commits/d791055d5584134e344c2edf6c48235f9f94d572 .